app/jobs/hearings/travel_board_hearing_sync_job.rb

```diff
@@ -95,14 +83,16 @@ def fetch_vacols_travel_board_appeals(ids, limit)
         bfdocind: nil,
         # Datetime of Decision
         bfddec: nil
-      ).limit(BATCH_LIMIT)
-    legacy_appeals = get_new_cases(cases, ids)
+      ).limit(BATCH_LIMIT).to_a
       .map do |vacols_case|
-        begin
-          AppealRepository.build_appeal(vacols_case, true)
-        rescue StandardError => error
-          log_error("#{error.class}: #{error.message} for vacols id:#{vacols_case.bfkey} on #{JOB_ATTR&.class} of ID:#{JOB_ATTR&.job_id}\n #{error.backtrace.join("\n")}")
-          next
+        # If there is not already a Legacy Appeal record tied to this case. Create one.
+        if!LegacyAppeal.where(vacols_id: vacols_case.bfkey).exists?
```

Comment:
This line looks wierd because theres no space in between the if condition
```suggestion
        if !LegacyAppeal.where(vacols_id: vacols_case.bfkey).exists?
```